src/membrane-keypad-element.ts

```diff
@@ -11,23 +20,33 @@ export class MembraneKeypadElement extends LitElement {
   @property() threeColumns = false;
 
   private pressedKeys = new Set<string>();
+  private shiftPressed = false;
 
   renderKey(text: string, x: number, y: number) {
     const keyClass = isNumeric(text) ? 'blue-key' : 'red-key';
+    let keyId = `key${text}`;
+    if (text === '*') {
+      keyId = 'keyAsterisk';
+    } else if (text === '#') {
+      keyId = 'keyHash';
+    }
```

Comment:
I wonder if there is a way to do this that won't break if we add a new, non-numeric key in the future. Low priority, but if you have an idea that doesn't require specific handling for Asterisk and Hash, that would be nice.

Reply:
I removed this part. It is problematic to use '#' in an id value, so now I used data attribute instead.